src/analyzers/refactoring-analyzer.ts

```
@@ -0,0 +1,353 @@
+import { FunctionInfo, RefactoringOpportunity } from '../types';
+
+export class RefactoringAnalyzer {
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)